x-pack/plugins/enterprise_search/public/applications/applications/components/playground/playground.tsx

```diff
@@ -11,11 +11,22 @@ import { useValues } from 'kea';
 
 import { i18n } from '@kbn/i18n';
 
+import type { PlaygroundPageMode } from '@kbn/search-playground/public';
+
 import { KibanaLogic } from '../../../shared/kibana';
 
 import { SearchPlaygroundPageTemplate } from './page_template';
 
-export const Playground: React.FC = () => {
+interface PlaygroundProps {
+  pageMode?: PageMode;
+}
+
+export enum PageMode {
```

Comment:
I think we should just use `PlaygroundPageMode` instead of introducing a duplicate type.

Reply:
This caused some issues with Kibana tooling, I will ask you offline if we can fix it properly as well. I am also not happy with this.